models/repo/update.go

```diff
@@ -19,19 +19,14 @@ func UpdateRepositoryOwnerNames(ctx context.Context, ownerID int64, ownerName st
 	if ownerID == 0 {
 		return nil
 	}
-	ctx, committer, err := db.TxContext(ctx)
```

Comment:
Again, nested transaction?

Reply:
There is no database write operation here, the transaction is unnecessary.